mappings/net/minecraft/world/gen/decorator/TreeDecorator.mapping

```diff
@@ -3,7 +3,7 @@ CLASS net/minecraft/class_4662 net/minecraft/world/gen/decorator/TreeDecorator
 	FIELD field_21319 type Lnet/minecraft/class_4663;
 	METHOD <init> (Lnet/minecraft/class_4663;)V
 		ARG 1 type
-	METHOD method_23469 generate (Lnet/minecraft/class_1936;Ljava/util/Random;Ljava/util/List;Ljava/util/List;Ljava/util/Set;Lnet/minecraft/class_3341;)V
+	METHOD method_23469 place (Lnet/minecraft/class_1936;Ljava/util/Random;Ljava/util/List;Ljava/util/List;Ljava/util/Set;Lnet/minecraft/class_3341;)V
```

Comment:
Should be `generate` to be consistent with `Feature`s.

Reply:
https://github.com/FabricMC/yarn/pull/1218/commits/587365eaae0d051be57d63b9eb21d28ef87e4c8f